src/Reservation.tsx

```diff
@@ -327,6 +335,28 @@ export default function Resevation(): JSX.Element {
                 {'  '}
               </p>
             </Grid>
+            <Grid item xs={12} className="center">
+              <Button
+                variant="contained"
+                className={classes.registerButton}
+                onClick={(): void => setThanks(false)}
+                startIcon={<EventAvailable />}
+                type="submit"
+              >
+                Make Another Reservation
+              </Button>
+            </Grid>
+            <Grid item xs={12}>
```

Comment:
Could this button also be centred?

Reply:
In doing so, it might also look nicer by decreasing the grid container's spacing

Reply:
Great feedback! Addressed!